src/events/scraper/run-scraper/scraper-helpers/get-schema-key-from-heading.js

```diff
@@ -0,0 +1,46 @@
+const assert = require('assert')
+const slugify = require('slugify')
+
+const slugifyOptions = { lower: true }
+
+const schemaKeys = [
+  'cases',
+  'hospitalized',
+  'deaths',
+  'recovered',
+  'state',
+  'county',
+  null // Use when we want to discard the column.
+]
+
+/**
+ * Hand-rolled version of _.pickBy from lodash/
+ * @param {object} object
+ * @param {(value:any, key: string|null) => boolean} predicate
+ */
+const pickBy = (object, predicate) => {
+  const obj = {}
+  for (const key in object) {
+    if (predicate(object[key], key)) {
+      obj[key] = object[key]
+    }
+  }
+  return obj
+}
+
+const assertAllValuesAreInSchema = (schemaKeysByHeadingFragment) => {
+  Object.values(schemaKeysByHeadingFragment).forEach(potentialKey => assert(schemaKeys.some(validKey => validKey === potentialKey), `Invalid value in schemaKeysByHeadingFragment: ${potentialKey}`))
```

Comment:
This line is 195 characters. Please don't make me add linting rules. :)

Can we pls break into multiple lines with clear actions?

Reply:
Can we please add linting rules? I'm so used to prettier at 120char that I don't normally have to think about line lengths 😬

Reply:
I was only kidding, but feel free to add! A hard 120 char would still get pretty far, but I still find it difficult to read, compared to something more verbose but comprehensible (to me) like:
```
const foundItems = pickBy(schemaKeysByHeadingFragment, (schemaKey, headingFragment) => {
  const heading = slugify(headingFragment, slugifyOptions)
  return slugHeading.includes(heading)
})
```